packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -166,6 +170,8 @@ export const prepareSettler = (
 ) => {
   assertAllDefined({ statusManager });
 
+  const isCctpDestSupported = makeIsCctpSupported(chainHub);
```

Comment:
this reads like a constant, not a lookup function. Maybe a reason to make this into an AddressHub or something now.
```
addressHub.supportsCctp(dest)
```